Test/MyPeptideTest.cs

```diff
@@ -209,13 +209,13 @@ public static void TestGoodPeptideWithLength()
         {
             var prot = new Protein("MNNNKQQQQMNNNKQQQQ", null);
 
-            DigestionParams digestionParams = new DigestionParams("Custom Protease7", maxMissedCleavages: 0, minPeptideLength: 1, initiatorMethionineBehavior: InitiatorMethionineBehavior.Retain);
+            DigestionParams digestionParams = new DigestionParams("trypsin", maxMissedCleavages: 0, minPeptideLength: 1, initiatorMethionineBehavior: InitiatorMethionineBehavior.Retain);
```

Comment:
no idea why these unit tests failed for me but worked elsewhere.. depends on what order the unit tests run I guess. the protease dictionary didn't have a "Custom protease7" but this test works fine w/ trypsin